### DISCUSSION TITLE
Per field error rather than whole request returning error?

### DISCUSSION THREAD

**Discussion by Niedzwiedzw** [async-graphql/async-graphql#1017] [Q&A]
I think juniper had this feature, where if say you had nested query that returned fallible data for range of objects 
```gql
parent {
    child {
       fallibleOperation
    }
}
```

if one of `fallibleOperation`s failed, I'd get that error for that specific field, but those `child`ren who succeded would have the data sent as usual. I see that in case of `async-graphql` you either get all the data or it returns first error it encounters. Is there any way to have the behaviour I described? thanks

**Comment by trkohler** [comment]
it must also be dependant of schema. If you have `fallibleOperation` as required field in schema, then on Error your query would fall up to the root. [why it is like that](https://graphql.org/learn/schema/#lists-and-non-null)
